torch/csrc/jit/type.cpp

```diff
@@ -25,8 +23,22 @@ std::ostream& operator<<(std::ostream & out, const Type & t) {
       }
     }
     out << ")";
-  TYPE_END()
+  } else if(t.kind() == TypeKind::HandleType) {
+    out << "Handle";
+  } else if(t.kind() == TypeKind::DynamicType) {
+    out << "Dynamic";
+  } else {
+    barf("unknown type kind");
+  }
   return out;
 }
 
+TypePtr HandleType::get() {
+  static auto value = std::make_shared<HandleType>();
+  return value;
+}
+TypePtr DynamicType::get() {
+  return std::make_shared<DynamicType>();
```

Comment:
Is there any reason why we treat `HandleType` as a singleton, but don't do the same for `DynamicType`?

Reply:
Nope. I fixed it.